lib/packwerk/configuration.rb

```diff
@@ -61,7 +61,7 @@ def initialize(configs = {}, config_path: nil)
     end
 
     def load_paths
-      @load_paths ||= ApplicationLoadPaths.extract_relevant_paths(@root_path, "test")
+      @load_paths ||= LoadPaths.extract_relevant_paths(@root_path, "test")
```

Comment:
```suggestion
      @load_paths ||= RailsLoadPaths.for(@root_path, environment: "test")
```

> I didn't update the API to LoadPaths.for(...), as I didn't see why we needed to create an instance and hold state here. Happy to change this if it's still desired.

I'm not sure I follow this comment. I meant for this line to work, so the signature would still be on the module and the environment would be a keyword. Does that make sense to you?

Reply:
Yes that makes sense to me! [This comment](https://github.com/Shopify/packwerk/pull/285#discussion_r1050106654) had me confused 
> I think it is better to name it LoadPaths.for(root, evironment:) or similar.
> It looks like we are exposing public methods we don't need to in it. extract_relevant_paths should be the only public method.

I thought you wanted the method to still be `extract_relevant_paths`, e.g. `LoadPaths.for(...).extract_relevant_paths`

I see what you're saying now though.